freqtrade/freqtradebot.py

```diff
@@ -1436,8 +1446,8 @@ def cancel_all_open_orders(self) -> None:
         Trade.commit()
 
     def handle_cancel_enter(
```

Comment:
@xmatthias

Reply:
I'm not entirely sure i see  / understand the problem? 
you'll always need a way to know WHICH order to cancel.

imagine this with true multi-order support (and the call from "replace-order") - you'll need to know if it's entry order A or B that you'd like to cancel.
i guess you could use the order object instead of the order-id - but that won't really make a difference, and will result in `order.order_id` instead of `order_id` from a usage perspective.